examples/rag/finetune.py

```diff
@@ -66,6 +70,25 @@ def __init__(self, *args, **kwargs):
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
 
+# https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/backends/test_accelerator_connector.py
+
+class Accel(DDPAccelerator):
```

Comment:
Why is this necessary?

Reply:
It also doesn't seem to be used anywhere

Reply:
Because of PTL DDP deprecation/migration to a different codepath in 1.0